HipoProject/urls.py

```diff
@@ -33,7 +33,8 @@
     path('api/profiles/me/change-password/', ChangePasswordView.as_view(), name='change-password'),
     path('api/ingredients/', ListCreateIngredientView.as_view(), name='list-create-ingredient'),
     path('api/recipes/', ListCreateRecipeView.as_view(), name='list-create-recipe'),
-    path('api/recipes/<int:pk>/', RecipeDetailView.as_view(), name='recipe-detail')
+    path('api/recipes/<int:pk>/', RecipeDetailView.as_view(), name='recipe-detail'),
+    path('api/recipes/<int:pk>/likes/', LikeDetailView.as_view(), name='like-recipe'),
```

Comment:
it is not `LikeDetailView`, we need to find a better name. What about`ListCreateDeleteLikesView` ?